test/test_sitemaps_from_robotstxt.py

```diff
@@ -0,0 +1,553 @@
+Disallow: /search/
+Disallow: /auth/
+Disallow: /auth.php
+Disallow: /personal/
+Disallow: /*?print=
+Disallow: /*&print=
+Disallow: /*register=yes
+Disallow: /*forgot_password=yes
+Disallow: /*change_password=yes
+Disallow: /*login=yes
+Disallow: /*logout=yes
+Disallow: /*auth=yes
+Disallow: /*action=ADD_TO_COMPARE_LIST
+Disallow: /*action=DELETE_FROM_COMPARE_LIST
+Disallow: /*action=ADD2BASKET
+Disallow: /*action=BUY
+Disallow: /*bitrix_*=
+Disallow: /*backurl=*
+Disallow: /*BACKURL=*
+Disallow: /*back_url=*
+Disallow: /*BACK_URL=*
+Disallow: /*back_url_admin=*
+Disallow: /*index.php$
+Disallow: /*?*
+
+
+User-agent: *
+Disallow: /bitrix/
+Disallow: /upload/
+Disallow: /detsad/
+Disallow: /humor/
+Disallow: /videouroki/
+Disallow: /radio/
+Disallow: /recepts/
+Disallow: /school_life/
+Disallow: /workgroups/
+Disallow: /institutions/
+Disallow: /kindergarten/
+Disallow: /unified-state-exam/
+Disallow: /ideas/
+Disallow: /documents/
+Disallow: /videosearch/
+Disallow: /auth/
+Disallow: /demotivators/
+Disallow: /additional-education/
+Disallow: /admission/
+Disallow: /random/
+Disallow: /horoscope/
+Disallow: /monitoring-in-the-sphere-of-education/
+Disallow: /votes/
+Disallow: /news/
+Disallow: /clips/
+Disallow: /preschool-education/
+Disallow: /movies/
+Disallow: /TV/
+Disallow: /dreambook/
+Disallow: /about/
+Disallow: /company/
+Disallow: /edit/
+Disallow: /com-docs/
+Disallow: /professional-education/
+Disallow: /vs.php
+Disallow: /index-old.php
+Disallow: /404.php
+Disallow: /suz/
+Disallow: /school-education/
+Disallow: /municipal-education-authorities/
+Disallow: /com-about/
+Disallow: /parents/
+Disallow: /view/
+Disallow: /stat/
+Disallow: /quotes/
+Disallow: /region/
+Disallow: /students/
+Disallow: /graduates/
+Disallow: /job/
+Disallow: /auth.php
+Disallow: /search/
+Disallow: /search/
+Disallow: /auth/
+Disallow: /auth.php
+Disallow: /personal/
+Disallow: /*?print=
+Disallow: /*&print=
+Disallow: /*register=yes
+Disallow: /*forgot_password=yes
+Disallow: /*change_password=yes
+Disallow: /*login=yes
+Disallow: /*logout=yes
+Disallow: /*auth=yes
+Disallow: /*action=ADD_TO_COMPARE_LIST
+Disallow: /*action=DELETE_FROM_COMPARE_LIST
+Disallow: /*action=ADD2BASKET
+Disallow: /*action=BUY
+Disallow: /*bitrix_*=
+Disallow: /*backurl=*
+Disallow: /*BACKURL=*
+Disallow: /*back_url=*
+Disallow: /*BACK_URL=*
+Disallow: /*back_url_admin=*
+Disallow: /*index.php$
+Disallow: /*?*
+Host: 3535.ru
+
+
+Sitemap: http://3535.ru/sitemap_000.xml
+'''.encode('utf-8')
+    )
+    job = SitemapExtractorJob()
+    job.init_accumulators(session=spark)
+    results = list(job.process_record(record))
+    assert len(results) == 1
+    assert results[0][0] == 'http://3535.ru/sitemap_000.xml'
+    assert results[0][1] == ['177.52.3535.ru']
+
+
+
+
+def test_host_accumulation_same_host(spark):
+    """
+    Test accumulation of hosts when sitemap url host and robots.txt url host match
+    Requires test/ on PYTHONPATH so utils._process_jobs can be imported
```

Comment:
A 4-line function could be inlined to avoid the import error.

However, `test/` is required on the path also to load `test_sitemaps_from_robotstxt` which is required for Spark serialization. So, you need to run it like
```
PYTHONPATH=$PYTHONPATH:./test python -m pytest test -v
```